modules/ztp-removing-content-from-managed-clusters.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-configuring-managed-clusters-policies.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-removing-content-from-managed-clusters_{context}"]
+= Removing content from a CR in managed clusters
```

Comment:
Think we could improve this title a bit. What about "Making changes to managed cluster CRs that are applied using policies"

or something similar...

Reply:
Changing applied managed cluster CRs using policies?